### ISSUE TITLE
[FEATURE]Add Scrollbar to Navigation Dropdown Menu for Improved Content Accessibility

### ISSUE THREAD

**Issue by k-vijay-05** [omegaup/omegaup#7926]
In the `navigation dropdown` section, the content is hidden if it exceeded the visible area, making it inaccessible to users.

**Before**: The dropdown menu content was hidden if it overflowed, and users were unable to view the hidden content.

https://github.com/user-attachments/assets/648055f8-f4e5-4638-acbe-68e044a3f94a

**After**:  The dropdown menu now includes a scrollbar, allowing users to view and access all the content that previously overflowed.

https://github.com/user-attachments/assets/254a5aad-f301-400a-b4a2-578071a5e69f

I would like to hear your suggestion on this change. Can I go ahead and raise a PR for this improvement?

**Comment by pabo99** [comment]
Hello @k-vijay-05, personally, I'm not a big fan of using scrollbars, but considering this bug might affect the user experience and may have gone unnoticed, your approach seems fine. Feel free to go ahead and create a new PR with the proposed solution.

**Comment by k-vijay-05** [comment]
@pabo99 Thank you for your feedback! I understand the preference for avoiding scrollbars. To align with this, I've updated the approach to include a hidden scrollbar while still enabling smooth scrolling for better user experience
